spec/latest/json-ld-api/index.html

```diff
@@ -504,7 +513,7 @@
     </pre>
 
     <p>Note how in the output above all properties of a <a>node</a> are collected in a
-      single <a>JSON object</a> and how the <a>blank node</a> representing
+      single <a class="changed">dictionary</a> and how the <a>blank node</a> representing
```

Comment:
I attempt to work the algorithms to deal just with the internal representation, and leave serialization to/from to the API wrapper.

I like the idea of describing the "JSON serialization" of the internal representation to clarify this. It's pretty equivalent to RDF Concepts use of N-Triples, which is not the same as the RDF Abstract Syntax.

I'll work on this some more.